scripts/dashboard/install-cephadm-e2e-deps.sh

```diff
@@ -15,21 +15,22 @@ on_error() {
 
 trap 'on_error $? $LINENO' ERR
 
-sudo apt -y install libvirt-daemon-system libvirt-daemon-driver-qemu qemu-kvm libvirt-clients
+# Install required deps.
+sudo apt update -y
+sudo apt install -y curl openssh-server
+if [[ ! $(command -v node) || $(node --version | grep -oE "v([0-9])+" | cut -c 2-) < 12 ]]; then
+    curl -fsSL https://deb.nodesource.com/setup_12.x | sudo -E bash -
```

Comment:
Can you use [this](https://github.com/nodesource/distributions/blob/master/README.md#manual-installation) method instead, please?

Please delete the `/etc/apt/sources.list.d/nodesource.list` file once install is complete too.

Reply:
@djgalloway Done!